tsconfig.json

```diff
@@ -40,5 +40,5 @@
     "src/css",
     "src/theme"
   ],
-  "include": ["./src/**/*.ts"]
+  "include": ["./src/**/*"]
```

Comment:
@ChocolateLoverRaj Let's keep this specific for now and just go for `ts` or `tsx`. We are in the process of converting all of the codebase to typescript. Then we will do `*`